Services/AdvancedMetaData/classes/class.ilAdvancedMDRecord.php

```diff
@@ -174,8 +174,15 @@ public static function _getAssignableObjectTypes($a_include_text = false)
 		$lng = $DIC['lng'];
 		
 		$types = array();
+		$filter = array();
 		$amet_types = $objDefinition->getAdvancedMetaDataTypes();
 
+		include_once './Services/WebServices/ECS/classes/class.ilECSSetting.php';
+		if(!ilECSSetting::_ecsConfigured()){
+			include_once './Services/WebServices/ECS/classes/class.ilECSEventQueueReader.php';
```

Comment:
includes are not required anymore, you can remove them.
for more information see: libs/composer/README.md (composer dump-autoload)